apps/workflows-dashboard/src/common/hooks/useLogout/useLogout.ts

```diff
@@ -0,0 +1,15 @@
+import { fetchLogout } from '@app/domains/auth/api/logout';
+import { sessionKeys } from '@app/domains/auth/api/session/query-keys';
+import { queryClient } from '@app/lib/react-query/query-client';
+import { useMutation } from '@tanstack/react-query';
+
+export function useLogout() {
```

Comment:
No need to change now but please use arrow functions moving forward unless it's something about `this` etc. 🙂